doc/ref/string.xml

```diff
@@ -150,7 +150,9 @@ doublequotes of a string literal to specify characters.
 They consist of two characters.
 The first is a backslash <C>\</C>. The second may be any character.
 If it is an octal digit (from <C>0</C> to <C>7</C>) there must be two more
-such digits. The meaning is given in the following list
+such digits.
+If it is the character <C>x</C>, then there must be two hexadecimal
+digits. The meaning is given in the following list
```

Comment:
This sounds weird, in particular the bit saying "then there must be two hexadecimal such digits", which seems to be missing something. Finally, the sentence "They consist of two characters." was already before this patch quite misleading. How about this:
```
There are a number of <E>special character sequences</E> that can be used
 between the singlequotes of a character literal or between the
 doublequotes of a string literal to specify characters.
They consists of a backslash <C>\</C>, followed by a second character indicating the type of
special character sequence, and, depending on this type, possibly some more characters.

The following special character sequencs are currently defined. Any other sequence starting with
a backslash results in an error.
```